src/model/immutable/ContentState.js

```diff
@@ -56,6 +56,17 @@ class ContentState extends ContentStateRecord {
     return block;
   }
 
+  getFirstLevelBlocks(): BlockMap {
+    return this.getBlockChildren('');
```

Comment:
Should we instead create a better way to define the root component? maybe a pre-defined key such as 'root' or something that could add a bit more meaning ?